samples/Management/PowerShell/AddingSchematoSchemagroups/Addingschematoschemagroup.ps1

```diff
@@ -0,0 +1,45 @@
+
+#Step 1 - Pass in AD Application details below to be able to fetch authorization token against legitimate Service Principal. 
+$Fields = @{
+    grant_type    = "client_credentials"
+    client_id     = "<Application_ID>"
+    resource      = "https://eventhubs.azure.net"
+    client_secret = "<Application_Secret>"
+  };
+
+  
+  $response = Invoke-RestMethod –Uri "https://login.microsoftonline.com/<Tenant_ID>/oauth2/token" –ContentType "application/x-www-form-urlencoded" –Method POST –Body $Fields

```

Comment:
For clarity, asssign the access token to a PS variable here instead of referencing $response in line 41.

Reply:
Ack'ed and updated it accordingly.